cmd/iac.go

```diff
@@ -0,0 +1,432 @@
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/rivo/tview"
+	"github.com/santiago-labs/telophasecli/lib/awsorgs"
+	"github.com/santiago-labs/telophasecli/lib/awssts"
+	"github.com/santiago-labs/telophasecli/lib/colors"
+	"github.com/santiago-labs/telophasecli/lib/telophase"
+	"github.com/santiago-labs/telophasecli/lib/ymlparser"
+)
+
+type iacCmd interface {
+	cdkCmd(result *sts.AssumeRoleOutput, acct ymlparser.Account, cdkPath string) *exec.Cmd
```

Comment:
this should just be one command and combined into one function on the interface. Since we can just take the path instead of cdkPath or tfPath. This makes it so we don't need that branching that you added below. That way we build the correct command before passing it in